control-plane/agents/src/bin/ha/cluster/switchover.rs

```diff
@@ -237,7 +237,8 @@ impl SwitchOverRequest {
                                 info!("Retrying Republish after cleaning up targets");
                                 return Ok(());
                             }
-                            _ => {
+                            Err(e) => {
+                                error!("error while listing nvme controller {}", e.to_string());
```

Comment:
instead, please add instrument at the top of the function:
#[tracing::instrument(level = "trace", skip(self), fields(volume.uuid = %self.volume_uuid), error]

Reply:
Done

Reply:
Actually use level = "info" as it'll be useful to see the spans on jaeger